sql/core/src/test/scala/org/apache/spark/sql/execution/debug/DebuggingSuite.scala

```diff
@@ -34,14 +34,13 @@ class DebuggingSuite extends SparkFunSuite with SharedSQLContext {
 
   test("debugCodegen") {
     val res = codegenString(spark.range(10).groupBy("id").count().queryExecution.executedPlan)
```

Comment:
can we change to `groupBy('id * 2)`? We should try our best to keep what to test, and keep 2 the shuffle in this query.

Reply:
Ok.